client/my-sites/upgrades/domain-management/controller.jsx

```diff
@@ -28,9 +29,9 @@ import titleActions from 'lib/screen-title/actions';
 const productsList = new ProductsList(),
 	sites = new SitesList();
 
-const renderPage = function( component ) {
+const renderPage = function( component, context ) {
 	ReactDom.render(
-		component,
+		React.createElement( Provider, { store: context.store }, component ),
```

Comment:
What's going on with all these `context` references? I don't think I fully understand. Could you explain a bit?